simple-bootstrap/3-operator-configs/gitops-job.yaml

```diff
@@ -0,0 +1,89 @@
+---
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  annotations:
+    example.com/managed-by: gitops
+    example.com/scm-url: git@github.com:redhat-cop/declarative-openshift.git
+  labels:
+    example.com/project: simple-bootstrap
+    example.com/component: namespaces
+  name: cronjob-gitops
+  namespace: cluster-ops
+spec:
+  concurrencyPolicy: Forbid
+  failedJobsHistoryLimit: 5
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          activeDeadlineSeconds: 500
+          containers:
+          - name: cronjob-gitops
+            command:
+            - /bin/bash
+            - -c
+            - |
+              if [ -z "$var" ]; then
+                echo "Syncing cluster config from $GIT_REPO/$CONTEXT_DIR"
+                git clone $GIT_REPO /tmp/repodir
+                until oc apply -Rf /tmp/repodir/$CONTEXT_DIR/ --prune -l $PRUNE_LABEL $(cat /tmp/repodir/prune-whitelist.txt); do sleep 2; done
```

Comment:
Can the API Group for DaemonSet be changed to `apps/v1` as `extensions/v1beta1` which causes the job to not complete in OCP 4.4